src/shared/modules/auth/common.js

```diff
@@ -184,6 +184,11 @@ export const temporarilyStoreUrlSearchParams = () => {
   )
 }
 
+export const getSSOServerIdIfShouldRedirect = () => {
```

Comment:
Maybe it's used in a later commit, but this function seems to be unused?

Reply:
Used now!